reference/Upload-API.yaml

```diff
@@ -1448,5 +1433,138 @@ paths:
+                      properties:
+                        id:
+                          type: string
+                        mediaData:
+                          type: object
+                          properties:
+                            mediaType:
+                              type: string
+                            entryId:
+                              type: string
+                            entryStatus:
+                              type: string
+                            thumbnailUrl:
+                              type: string
+                        metaData:
+                          type: object
+                          additionalProperties: true
+                  pagingData:
+                    type: object
+                    properties:
+                      totalCount:
+                        type: integer
+                      itemCount:
+                        type: integer
+                      pageCount:
+                        type: integer
+                      pageSize:
+                        type: integer
+                      pageIndex:
+                        type: integer
+                x-examples:
+                  Example 1:
+                    metaData:
+                      id: d70db4a4-f454-428b-a2fe-df37950a6c35
+                      name: test_feed
+                      title: Test feed
+                      description: null
+                      target: Unrestricted
+                    itemData:
+                      - id: f7669a91-f4df-4dc0-950c-94bd087a9d33
+                        mediaData:
+                          mediaType: OnDemand
+                          entryId: 0_1o4osi6n
+                          entryStatus: Ready
+                          thumbnailUrl: 'https://open.http.mp.streamamg.com/p/3000759/sp/300075900/thumbnail/entry_id/0_1o4osi6n/'
+                        metaData:
+                          home_team: null
+                          body: ''
+                          home_players: []
+                          multiselectdd: []
+                          opta_fixture: null
+                          content_name: []
+                          away_players: []
+                          competition: null
+                          content_type: []
+                          away_team: null
+                          livescoreplayers: []
+                          sysentryentitlements: []
+                          ground: null
+                          tournaments: []
+                          country: null
+                          createdat: null
+                          VideoDuration: 183
+                          optaseriesdropdown: null
+                          opta_season: null
+                          title: sample_1280x720_surfing_with_audio
+                          episode: null
+                          tags:
+                            - ''
+                        publicationData:
```

Comment:
The date time formats seem a little inconsistent i.e `2022-11-25T16:34:19` vs `999-12-31T23:59:59.9999999` and are missing timezone info. i.e `2022-11-25T16:34:19Z`

Reply:
Checking

Reply:
The date '9999-12-31T23:59:59.9999999' is the default maximum date, that's the response by the API for now, 
If any correction will be done in CM